src/viam/sdk/common/utils.hpp

```diff
@@ -18,30 +18,28 @@ const std::string kService = "service";
 const std::string kRDK = "rdk";
 const std::string kBuiltin = "builtin";
 
+using time_pt = std::chrono::time_point<std::chrono::system_clock, std::chrono::nanoseconds>;
```

Comment:
What motivated the switch from calling it `time_point` to `time_pt`?

Reply:
I find people do 'using namespace std::chrono' pretty often so I think this would be a name clash with our `time_point` and `std::chrono::time_point<Clock, Duration>`